qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1522,6 +1522,25 @@ def quantum_predecessors(self, node):
             )
         )
 
+    def classical_predecessors(self, node):
+        """Returns iterator of the predecessors of a node that are
+        connected by a classcical edge as DAGOpNodes and DAGInNodes."""
+        return iter(
+            self._multi_graph.find_predecessors_by_edge(
+                node._node_id, lambda edge_data: isinstance(edge_data, Clbit)
+            )
+        )
+
+    def classical_successors(self, node):
```

Comment:
I think it would be a bit tidier to move this `classical_successors` method further down, just after the `quantum_successors` method